<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1809</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1741</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>出行小助手</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>340</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>1001</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>智能导航</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>80</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Operation1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>操作1</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Johnson</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Floyd-Warshall</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起点</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>终点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询起点到终点的最短路径</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>操作2</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询从起点到其他点的最短路径</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Floyd-Warshall</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Johnson</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起点</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>操作3</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>地铁线路1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Krustal</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prim</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>地铁线路2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>操作4</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>501</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询从起点出发的最佳公交路线</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起点</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
